### PR TITLE
Fb if for

### PR DESCRIPTION
**Pull request by kingwangzzang1234**

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,16+1):
+    if i%15==0:
```
Comment (ulgoon) [inline fizzbuzz.py:2]: Could you write this statements in 1 if?

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        print('fizzbuzz')
+    elif i%3==0:
+        print('fizz')
+    elif i%5==0:
+        print('buzz')
+    else:
+        print(i)
```
Comment (ulgoon) [inline fizzbuzz.py:9]: print done! if for statement finished

**Review by ulgoon — CHANGES_REQUESTED** [review]

File: fizzbuzz.py (hunk 1 above)
Comment (kingwangzzang1234) [reply fizzbuzz.py:2]: Urgh.... got it.

File: fizzbuzz.py (hunk 2 above)
Comment (kingwangzzang1234) [reply fizzbuzz.py:9]: No problem

**Review by ulgoon — APPROVED** [review]

**Pull request merged**